src/periodic_table.py

```diff
@@ -0,0 +1,359 @@
+METAL = 'Metal'
+NON_METAL = 'Non Metal'
+METALLOID = 'Metalloid'
+
+PERIODIC_TABLE = {
+    '1': {'atomic_num': 1, 'symbol': 'H', 'name': 'Hydrogen', 'group': 1, 'period': 1, 'atomic_weight': 1.008,
+          'density': 8.988e-05, 'melting_point': 14.01, 'boiling_point': 20.28, 'heat_capacity': 14.304,
+          'electronegativity': 2.2, 'precise': True, 'type': NON_METAL},
+    '2': {'atomic_num': 2, 'symbol': 'He', 'name': 'Helium', 'group': 18, 'period': 1, 'atomic_weight': 4.002602,
+          'density': 0.0001785, 'melting_point': None, 'boiling_point': 4.22, 'heat_capacity': 5.193,
+          'electronegativity': None, 'precise': True, 'type': NON_METAL},
+    '3': {'atomic_num': 3, 'symbol': 'Li', 'name': 'Lithium', 'group': 1, 'period': 2, 'atomic_weight': 6.94,
+          'density': 0.534, 'melting_point': 453.69, 'boiling_point': 1560, 'heat_capacity': 3.582,
+          'electronegativity': 0.98, 'precise': True, 'type': METAL},
+    '4': {'atomic_num': 4, 'symbol': 'Be', 'name': 'Beryllium', 'group': 2, 'period': 2, 'atomic_weight': 9.0121831,
+          'density': 1.85, 'melting_point': 1560, 'boiling_point': 2742, 'heat_capacity': 1.825,
+          'electronegativity': 1.57, 'precise': True, 'type': METAL},
+    '5': {'atomic_num': 5, 'symbol': 'B', 'name': 'Boron', 'group': 13, 'period': 2, 'atomic_weight': 10.81,
+          'density': 2.34, 'melting_point': 2349, 'boiling_point': 4200, 'heat_capacity': 1.026,
+          'electronegativity': 2.04, 'precise': True, 'type': METALLOID},
+    '6': {'atomic_num': 6, 'symbol': 'C', 'name': 'Carbon', 'group': 14, 'period': 2, 'atomic_weight': 12.011,
+          'density': 2.267, 'melting_point': 3800, 'boiling_point': 4300, 'heat_capacity': 0.709,
+          'electronegativity': 2.55, 'precise': True, 'type': NON_METAL},
+    '7': {'atomic_num': 7, 'symbol': 'N', 'name': 'Nitrogen', 'group': 15, 'period': 2, 'atomic_weight': 14.007,
+          'density': 0.0012506, 'melting_point': 63.15, 'boiling_point': 77.36, 'heat_capacity': 1.04,
+          'electronegativity': 3.04, 'precise': True, 'type': NON_METAL},
+    '8': {'atomic_num': 8, 'symbol': 'O', 'name': 'Oxygen', 'group': 16, 'period': 2, 'atomic_weight': 15.999,
+          'density': 0.001429, 'melting_point': 54.36, 'boiling_point': 90.2, 'heat_capacity': 0.918,
+          'electronegativity': 3.44, 'precise': True, 'type': NON_METAL},
+    '9': {'atomic_num': 9, 'symbol': 'F', 'name': 'Fluorine', 'group': 17, 'period': 2, 'atomic_weight': 18.998403163,
+          'density': 0.001696, 'melting_point': 53.53, 'boiling_point': 85.03, 'heat_capacity': 0.824,
+          'electronegativity': 3.98, 'precise': True, 'type': NON_METAL},
+    '10': {'atomic_num': 10, 'symbol': 'Ne', 'name': 'Neon', 'group': 18, 'period': 2, 'atomic_weight': 20.1797,
```

Comment:
What corresponds to period and what corresponds to group in our case?

Reply:
Period = row
Group = column